### ISSUE TITLE
Fix travis.yaml

### ISSUE THREAD

**Issue by mariobehling** [fossasia/loklak_search#8]
[31;1mERROR:[0m An error occured while trying to parse your .travis.yml file.

Please make sure that the file is valid YAML.

http://lint.travis-ci.org can check your .travis.yml.

The log message was: Build config file had a parse error: found character that cannot start any token while scanning for the next token at line 10 column 1.

**Comment by jig08** [comment]
**Travis WebLint** Result: 

- value for addons section is empty, dropping
- in addons section: unexpected key apt, dropping



<img width="1280" alt="screen shot 2016-10-03 at 6 20 37 pm" src="https://cloud.githubusercontent.com/assets/9291400/19037841/1edfdcd6-8996-11e6-9aa2-2bf709e0dd7b.png">

**Comment by hemantjadon** [comment]
![screen shot 2016-10-07 at 12 47 48 pm](https://cloud.githubusercontent.com/assets/11755543/19181691/5d162962-8c8c-11e6-8433-3456ffca57f0.png)

* The travis build is working fine on the fork, but the travis.yml shows errors at http://lint.travis-ci.org/

**Comment by jig08** [comment]
@hemantjadon 
This is the exact error in `.travis.yml`:
```
> value for addons section is empty, dropping
> in addons section: unexpected key apt, dropping
```

<img width="1280" alt="screen shot 2016-10-07 at 12 55 27 pm" src="https://cloud.githubusercontent.com/assets/9291400/19181845/667a5a68-8c8d-11e6-8856-960a286607d9.png">

**Comment by hemantjadon** [comment]
@jig08 
I know that error but what i want to say is i don't know why the travis is building and running tests in the fork perfectly

**Comment by jig08** [comment]
@hemantjadon 

>Does dropping mean it is ignoring those keys ??

Yes.

**Comment by hemantjadon** [comment]
Thanks @jig08
I will lint this asap.

**Comment by hemantjadon** [comment]
This issue can be closed now
Resolved in #10

**Comment by mariobehling** [comment]
Build still shows up as "unknown" on Readme.md.
